autoPyTorch/pipeline/components/setup/network_backbone/ShapedMLPBackbone.py

```diff
@@ -125,10 +124,15 @@ def get_hyperparameter_search_space(
 
         # We can have dropout in the network for
         # better generalization
+        dropout_flag = False
+        if any(use_dropout.value_range):
+            dropout_flag = True
```

Comment:
I am wondering if we really need `use_dropout`, because it just increases the dimensionality and does not see any benefits from this.
Basically, when the observations accumulate, SMAC itself knows if we really need dropout or not.
If we do not really need it, SMAC just starts to take lower dropout values.